.github/workflows/test.yml

```diff
@@ -60,6 +61,7 @@ jobs:
         run: |
           chmod 0600 $PGPASSFILE
           db-setup/create-db.sh
+          docker restart --time 0 sr_test_db
```

Comment:
whats the point of this docker cmd? I can see it restarts the sr_test_db container, not sure why, also I thought we werent using docker for the db in ci

Reply:
A restart is necessary due to the change made in `reset-db.sql`, which increases the number of connections allocated to the database. That setting does not take effect until the database is forcibly restarted.

We are using docker for the database in the testing flow.

Reply:
In that case, it makes more sense to me to just start the container with the correct [max-connections set](https://hub.docker.com/_/postgres#:~:text=postgres%20%2Dc%20shared_buffers%3D256MB%20%2Dc%20max_connections%3D200), 

If the github actions magical syntax doesn't support the passing of arguments, can we just add a separate step for running the docker command directly at the beginning of the steps?

`docker run -d -p 5432:5432 --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -e POSTGRES_PASSWORD=this_is_a_super_secure_admin_password postgres:13-alpine3.16 -c max_connections=500`

Reply:
The benefit of doing it this way is that we could set the max value in the `reset-db` script (which `create-db.sh` uses) and that max value is used both here and in the docker test setup (used for local testing). If we change the docker run command directly here, we'd still need to change the value for local docker-compose. There was a prior commit that used that approach (back when we were testing locally and hadn't yet figured out that CI is different), [4a1ef269](https://github.com/CDCgov/prime-simplereport/pull/4722/commits/4a1ef269a6ea4f6b085c60056898add9ca905656), but I prefer this way because it keeps the local test setup and the CI setup more in alignment.

The real solution would be to use the full docker-compose script in CI instead of this custom action, but that's out of scope for today 😅

Reply:
I agree about the docker-compose solution, can you please write a ticket to document and implement that solution, the current solution is certainly a half measure

Reply:
Done: https://github.com/CDCgov/prime-simplereport/issues/4856